packages/tasks/src/model-libraries.ts

```diff
@@ -1170,6 +1170,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path_extension:"pt"`,
 		snippets: snippets.vui,
 	},
+	vibevoice: {
+		prettyLabel: "VibeVoice",
+		repoName: "VibeVoice",
+		repoUrl: "https://github.com/microsoft/VibeVoice",
+		snippets: snippets.vibevoice,
+		filter: false,
+		countDownloads: `path:"pytorch_model.bin" OR path:"model.safetensors"`,
```

Comment:
From what I can see in https://huggingface.co/aoi-ot/VibeVoice-1.5B/tree/main https://huggingface.co/aoi-ot/VibeVoice-7B/tree/main https://huggingface.co/aoi-ot/VibeVoice-Large/tree/main, non of the repos have `pytorch_model.bin` or `model.safetensors` files.

Since they all have a `config.json` file, I think it's best to count downloads based on that. Since it's the default behavior we can just drop the `countDownload` line

```suggestion
```